website/pages/partials/envvars.mdx

```diff
@@ -81,12 +81,24 @@
       </td>
       <td>Group&lsquo;s name</td>
     </tr>
+    <tr>
+      <td>
+        <code>NOMAD_JOB_ID</code>
```

Comment:
We should add this in https://github.com/hashicorp/nomad/blob/master/website/pages/docs/runtime/environment.mdx#task-identifiers as well. I feel like we should probably differentiate between the job ID and the job name here in the documentation... if you're submitting jobs via `nomad job run` then it's not obvious what the difference would be, because they're only different for jobs submitted via the HTTP API.

Reply:
Good pointers, added that. I extended the wording of the Job ID to make that difference clear - do you think that works?

Reply:
Yup, that looks good.